js/src/components/ads-account-select-control/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import AppSelectControl from '.~/components/app-select-control';
+import useExistingGoogleAdsAccounts from '.~/hooks/useExistingGoogleAdsAccounts';
+
+/**
+ * @param {Object} props The component props
+ * @param {string} [props.value] The selected value. IF no value is defined, then the first option is selected and onChange function is triggered.
+ * @param {Function} [props.onChange] Callback when the select value changes.
+ * @return {JSX.Element} An enhanced AppSelectControl component.
+ */
+const AdsAccountSelectControl = ( {
+	value,
+	onChange = () => {},
+	...props
+} ) => {
+	const { existingAccounts: accounts = [] } = useExistingGoogleAdsAccounts();
+
+	const options = accounts.map( ( acc ) => ( {
+		value: acc.id,
+		label: sprintf(
+			// translators: 1: account name, 2: account ID.
+			__( '%1$s (%2$s)', 'google-listings-and-ads' ),
+			acc.name,
+			acc.id
+		),
```

Comment:
```suggestion
		label: `${ acc.name } (${ acc.id })`,
```

Can we simplify like this?

Reply:
Sure thing @ankitrox . Updated!